sycl/test/regression/cache_test.cpp

```diff
@@ -0,0 +1,148 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -I %sycl_include/../ %s -o %t.out -L %sycl_libs_dir -lze_loader
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
```

Comment:
It seems the test requires Level zero implementation on the system. If so, could you please add "// REQUIRES: level_zero"

Reply:
For some reason, when I add "// REQUIRES: level_zero" to the test, the test becomes "UNSUPPORTED" and states that the "Test requires the following unavailable features: level_zero". Is this expected behavior? I receive the same output from all of the lit tests that contain "// REQUIRES: level_zero". 

The test works fine without this line, could this be an environment issue?

@glyons-intel I see that you have added a similar line in one of the lit tests. Do you see any issues when using "// REQUIRES: level_zero"?

Reply:
Could you please clarify how you run this test?
The test can be working without REQUIRES if you have Level Zero implementation on your system.

Reply:
I have been running this test with the following command:

llvm-lit -a -vv cache_test.cpp

from within the test/regression directory.

Reply:
I am seeing the same behavior as Kieran.
-- Testing: 1 tests, 1 workers --
UNSUPPORTED: SYCL :: usm/dmem_varied.cpp (1 of 1)

Testing Time: 0.11s
  Unsupported: 1

When I removed the "// REQUIRES: level_zero",  it passes.

Reply:
Thanks @gmlueck. When adding the --param SYCL_BE=PI_LEVEL_ZERO as an option to my llvm-lit command, the test runs successfully. @romanovvlad, I have updated the test to require a level zero backend ("// REQUIRES: level_zero").

Reply:
Thanks!